data/building-hours.json

```diff
@@ -87,86 +164,210 @@
         }
     },
     {
-        "name": "Skoglund",
-        "image": "skoglund",
+        "name": "DiSCO",
+        "image": "disco",
         "times": {
             "hours": {
-                "Mon": ["6:30", "22:45"],
-                "Tue": ["6:30", "22:45"],
-                "Wed": ["6:30", "22:45"],
-                "Thu": ["6:30", "22:45"],
-                "Fri": ["6:30", "20:45"],
-                "Sat": ["9:00", "20:45"],
-                "Sun": ["12:00", "22:45"]
+              "Mon": ["10:00", "17:00"],
+              "Tue": ["10:00", "17:00"],
+              "Wed": ["10:00", "17:00"],
+              "Thu": ["10:00", "17:00"],
+              "Fri": ["10:00", "17:00"],
+              "Sun": ["13:00", "17:00"]
             }
         }
     },
     {
-        "name": "The Cage",
-        "image": "cage",
-        "notes": "The kitchen stops cooking at 8 p.m.",
+        "name": "Alumni Hall",
+        "image": "alumni",
         "times": {
             "hours": {
-                "Mon": ["7:30", "23:00"],
-                "Tue": ["7:30", "23:00"],
-                "Wed": ["7:30", "23:00"],
-                "Thu": ["7:30", "23:00"],
-                "Fri": ["7:30", "20:00"],
-                "Sat": ["9:00", "20:00"],
-                "Sun": ["9:00", "23:00"]
+              "Mon": ["7:00", "17:00"],
+              "Tue": ["7:00", "17:00"],
+              "Wed": ["7:00", "17:00"],
+              "Thu": ["7:00", "17:00"],
+              "Fri": ["7:00", "17:00"]
             }
         }
     },
     {
-        "name": "Stav Hall Breakfast",
-        "image": "stav",
+        "name": "Boe Chapel",
+        "image": "boe",
         "times": {
             "hours": {
-              "Mon": ["7:00", "9:45"],
-              "Tue": ["7:00", "9:45"],
-              "Wed": ["7:00", "9:45"],
-              "Thu": ["7:00", "9:45"],
-              "Fri": ["7:00", "9:45"],
-              "Sat": ["7:00", "9:45"],
-              "Sun": ["8:30", "10:15"]
+              "Mon": ["7:00", "24:00", {"nextDay": true}],
+              "Tue": ["7:00", "24:00", {"nextDay": true}],
+              "Wed": ["7:00", "24:00", {"nextDay": true}],
+              "Thu": ["7:00", "24:00", {"nextDay": true}],
+              "Fri": ["7:00", "21:30"],
+              "Sat": ["8:00", "21:30"],
+              "Sun": ["8:00", "24:00", {"nextDay": true}]
```

Comment:
It works this way too, I think it's not a big deal, I'd be ok either way. I think that the `24:00` is a little more descriptive for the programmer, semantically since it's larger than the `8:00`. I'll leave it up to you, @zimmer18